### ISSUE TITLE
Unrecognized Locales prevent App Store submission (Regression)

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-ios#845]
ITMS-90176: Unrecognized Locale - - The locale names used in localization directories at ( 'Payload/OpenFoodFacts.app/ber.lproj', 'Payload/OpenFoodFacts.app/me.lproj', 'Payload/OpenFoodFacts.app/ry.lproj', 'Payload/OpenFoodFacts.app/son.lproj', 'Payload/OpenFoodFacts.app/tzl.lproj' ) are invalid

**Comment by aleene** [comment]
Yes you should delete those from Crowdin, etc. We have noticed this issue before.

**Comment by aleene** [comment]
- #736

**Comment by teolemon** [comment]
@aleene How about adding them to .gitignore ?
https://github.com/openfoodfacts/openfoodfacts-ios/blob/develop/.gitignore
But meanwhile, I believe I disabled them in Crowdin, so we can probably close:
